controllers/topic.go

```diff
@@ -637,6 +642,96 @@ func (c *ApiController) EditContent() {
 	c.ServeJSON()
 }
 
+func (c *ApiController) TranslTopic() {
```

Comment:
Why don't use "Translate" everywhere?

Reply:
it's too long

Reply:
I think that's OK. `Transl` doesn't look good.

Reply:
fixed